Makefile

```diff
@@ -142,6 +142,6 @@ push: docker-image-build
 	$(DOCKER_PUSH) $(call unescape_docker_tag,$(DOCKER_IMAGE_VERSIONED))
 	@if [ "$$TRAVIS_TAG" != "" ]; then \
 		$(DOCKER_TAG) $(call unescape_docker_tag,$(DOCKER_IMAGE_VERSIONED)) \
-			$(call unescape_docker_tag,$(DOCKER_IMAGE)):latest
-		$(DOCKER_PUSH) $(call unescape_docker_tag,$(DOCKER_IMAGE):latest)
+			$(call unescape_docker_tag,$(DOCKER_IMAGE)):latest; \
```

Comment:
I'm not string against it either, just commenting the usual way to write those commands in Dockerfiles. Your arguments make sense and the only advantage of the `&&` would be that the script would fail at the specific point of failure (and not one command later) so leave it with the `;` if you want.